lib/tasks/report_duplicate_organizations.rake

```diff
@@ -0,0 +1,17 @@
+namespace :report do
+  desc 'Print a count and list of domains with possible duplicate organizations'
+  task duplicate_organizations: :environment do
+    domains = Website.pluck(:content, :organization_id).group_by do |item|
+      content, _id = item
+      content.split('/')[2]
+    end
+
+    duplicates = domains.select do |_website, grouped_results|
+      next false unless grouped_results.size > 1
+      grouped_results.map(&:last).uniq.size > 1
+    end
+
+    puts "Count: #{duplicates.values.map(&:last).flatten.uniq.count}"
+    puts duplicates
+  end
```

Comment:
Changed the names around here, but it's otherwise the same from yesterday when we wrote it.